packages/api-reference/src/features/BaseUrl/ServerVariablesSelect.vue

```diff
@@ -36,9 +36,9 @@ const selected = computed<ScalarListboxOption | undefined>({
       class="variable-select"
       fullWidth
       variant="ghost">
-      <span>
+      <span :class="{ 'text-c-1': value }">
         <span class="sr-only">Selected:</span>
```

Comment:
We can probably hide this screen reader text if `value` isn't set, otherwise it will read "Selected: Select Value" (and it can probably just say "Select Value")

Reply:
oh great catch!